### ISSUE TITLE
ngt 1.13.0 build failure

### ISSUE THREAD

**Issue by chenrui333** [yahoojapan/NGT#89]
👋 trying to build the latest release, but run into some build issue. The error log is as below:

<details>
<summary>build failure</summary>

```
/tmp/ngt-20210115-42869-please/NGT-1.13.0/lib/NGT/NGTQ/QuantizedGraph.h:301:23: error: use of undeclared identifier '_MM_HINT_T0'
                _mm_prefetch(&o[0], _MM_HINT_T0);
                                    ^
```

</details>

Full build log is in here, https://github.com/Homebrew/homebrew-core/runs/1705868068
relates to https://github.com/Homebrew/homebrew-core/pull/69037

**Comment by chenrui333** [comment]
trying the ngt 1.13.1 in Homebrew/homebrew-core#69037

**Comment by masajiro** [comment]
I found that the same issue occured when avx is unavailable. This issue I found was fixed in the 1.13.1. Did the 1.13.1 resolve your issue?

**Comment by chenrui333** [comment]
yeah, the build is green after 1.13.1. thanks @masajiro for checking in.

closing the issue.
